.github/workflows/trivy.yml

```diff
@@ -0,0 +1,91 @@
+
+name: trivy
+on:
+  # runs on default branch
+  schedule:
+    - cron:  '0 22 * * 0' # each Monday at 9am AEST+10 / 10am AEDT+11
+
+jobs:
+  trivy-prod:
+    name: Trivy check
+    runs-on: ubuntu-latest
+
+    env:
+      # BUILDKIT_PROGRESS: plain
+      CLOUDSDK_CORE_DISABLE_PROMPTS: 1
+      DOCKER_IMAGE: australia-southeast1-docker.pkg.dev/seqr-308602/seqr-project/seqr
```

Comment:
Nit: this isn't really the full Docker image path. How about pulling up the `gcloud-prod` from below, as there's a separate job for `gcloud-dev` anyhow?

Reply:
Sounds good, resolved!